docassemble/EFSPIntegration/efm_client.py

```diff
@@ -180,121 +186,205 @@ def RemoveUser(self, id:str):
 
     # TODO(brycew): not tested
     def ResetUserPassword(self, email:str):
-      resp = self.proxy_client.post(self.base_url + 'adminusers/user/reset_password', data=email)
+      resp = self.proxy_client.post(self.base_url + 'adminusers/users/reset_password', data=email)
       return ProxyConnection.user_visible_resp(resp)
 
     # Managing a Firm
     def GetFirm(self, firm_id):
-      resp = self.proxy_client.get(self.base_url + 'firmattorneyservice/firm')
+      # TODO: QS changed url to be more consistent--double-check please!
+      resp = self.proxy_client.get(self.base_url + f'firmattorneyservice/firms/{firm_id}')
```

Comment:
This is another weird one, as you can only ever get the firm associated with your user (if there is one). There's no SOAP API on Tyler's side that let's you get a firm using an arbitrary id. 

Idk if it makes sense, but I was trying to just URLs that get you info about the authenticated user. I'm realizing now that that's not really stateless, so it's probably not the best. But for things like firms, we can't get any other information about other firms anyway.